tables/bg-6dots.dis

```diff
@@ -0,0 +1,84 @@
+# liblouis: bg-6dots.dis
```

Comment:
I wonder what the intended output encoding of this display table is? Latin-5?

By the way why not call the file "bg.dis"? Is there an alternative version?

Reply:
> intended output encoding

I meant the "intended _character set_". (Encoding shouldn't be done by a display table, that's the responsibility of the application that uses Liblouis.)

Reply:
OK, I renamed the display table